src/Test/TestCombatMods.java

```diff
@@ -98,7 +95,7 @@ private boolean testRetribution()
     BattleSummary normalAB = CombatEngine.simulateBattleResults(infA, infB, testMap, 7, 4);
     BattleSummary normalBA = CombatEngine.simulateBattleResults(infB, infA, testMap, 7, 4);
 
-    venge.modifyAbilityPower(42); // juice up
+    venge.modifyAbilityPower(Commander.CHARGERATIO_FUNDS*42); // juice up
```

Comment:
I'm not a fan of needing to know to invoke `CHARGERATIO_FUNDS` to turn a number into a power level.